NineChronicles.Standalone/Controllers/GraphQLController.cs

```diff
@@ -107,6 +110,30 @@ public IActionResult SetMining([FromBody] SetMiningRequest request)
             return Ok($"Set mining status to {request.Mine}.");
         }
 
+        [HttpPost(CheckPeerEndpoint)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<IActionResult> CheckPeer([FromBody] CheckPeerRequest request)
+        {
+            try
+            {
+                var exist = await StandaloneContext.NineChroniclesNodeService.CheckPeer(request.AddressString);
+                if (exist)
+                {
+                    return Ok($"Found peer {request.AddressString}.");
+                }
+                
+                return BadRequest($"No such peer {request.AddressString}");
+            }
+            catch (Exception e)
+            {
+                var msg = $"Unexpected error occurred during CheckPeer request. {e}";
+                // Unexpected Error.
+                Log.Warning(e, msg);
+                return BadRequest(msg);
```

Comment:
In my thought, it will match `InternalServerError` than `BadRequest`. Is there `InternalServerError` method in `ControllerBase`? I couldn't find it in its document.

Reply:
I've added your suggestion in [`aa4161a`](https://github.com/planetarium/NineChronicles.Standalone/pull/32/commits/aa4161a23f2983a96d27a868f066ea528686dd70)